packages/webdriverio/tests/utils/interception/devtools.test.js

```diff
@@ -156,7 +156,7 @@ test('decodes base64 responses', async () => {
     const mock = new NetworkInterception('**/foobar/**')
     await NetworkInterception.handleRequestInterception(cdpClient, [mock])({
         request: { url: 'http://test.com/foobar/test.html' },
-        responseHeaders: [{ name: 'Content-Type', value: 'application/json' }]
+        responseHeaders: [{ name: 'content-Type', value: 'application/json' }]
```

Comment:
I assume this should be `content-type`, given that the case changed in the other place as well?

Reply:
It's test, wanted to have this one in mixed case

Reply:
Oh, of course, thank you!